llvm/lib/Transforms/IPO/AttributorAttributes.cpp

```diff
@@ -12987,6 +12989,225 @@ struct AAAllocationInfoCallSiteArgument : AAAllocationInfoImpl {
 };
 } // namespace
 
+/// ----------- AAConvertOutArgument ----------
+namespace {
+static bool isEligibleArgument(const Argument &Arg, Attributor &A,
+                               const AbstractAttribute &AA) {
```

Comment:
Make this a (static) member function, no need for the global namespace. If it's a regular member you don't need to pass AA.